src/sections/Home/Proud-maintainers/proudMaintainers.style.js

```diff
@@ -5,8 +5,19 @@ const ProjectItemWrapper = styled.section`
 		align-self: flex-start;
 		margin-left: -5rem;
 		padding: 0rem;
+		
+		@media screen and (max-width: 1600px) {
+			margin-left: 5rem;
+		}
 		@media screen and (max-width: 992px) {
-			margin-left: 0rem;
+			margin-left: 4rem;
+		}
+		@media screen and (max-width: 768px) {
+			margin-left: 2rem;
+
+		}
+		@media screen and (max-width: 550px) {
+			margin-left: 3rem;
```

Comment:
use 0.3 rem

Reply:
@legendarykamal Thanks please merge this PR